docs/tutorials/change-date-on-a-task

```diff
@@ -0,0 +1,104 @@
+# Tutorial: Change the due date on a task
+
+In this tutorial, you will learn how to change the due date on a task.
+
+Expect this tutorial to take about 15 minutes to complete.
+
+## Before you start
+
+Make sure you have completed the [Before you start a tutorial](before-you-start-a-tutorial) topic on the development system you will use for this tutorial.
+
+## Find an existing task
+
+Unless you already know the task's unique ID, you must first retrieve a list of tasks. To do this, use the GET method for {base_url}/tasks. 
+
+View a list of all tasks
+
+1. If your local service is not running, start it.
+
+    ```shell
+    cd <your-github-workspace>/to-do-service/api
+    # Run the service and monitor its database file for updates
+    json-server -w to-do-db-source.json
+    ```
+
+1. On your desktop, open the Postman app.
+1. In the Postman app, create a new request with the values below.
+    * **METHOD**: GET
+    * **URL**: `{base_url}/tasks`
+    * **Headers**:
+        * `Content-Type: application/json`
+    * **Request body**:
+       None
+
+1. In the Postman app, choose **Send** to make the request.
+1. Watch for the response body, which should look similar to the text below. Make note of the `id` for the task (or tasks) you want to change.
+
+  
+```js
+[
```

Comment:
This abbreviated notation might be confusing.
It might be better to include an example of two complete tasks and an ellipsis to imply that there are more, than abbreviating the task resources.

Reply:
I did not do that on purpose because I have no idea what I'm doing :)